packages/compass-aggregations/src/components/stage-preview/stage-preview.module.less

```diff
@@ -66,26 +66,36 @@
     display: flex;
     align-items: stretch;
     overflow-x: scroll;
-    margin: 10px;
-    padding: 10px;
+    padding: 16px;
   }
 
-  &-document-card {
+  &-document-card-container {
     flex: none;
     border: 1px solid @gray6;
     border-radius: 4px;
     box-shadow: 1px 1px 4px rgba(0, 0, 0, 0.1);
     width: 350px;
-    height: 150px;
     overflow: scroll;
+    min-height: 150px;
+    flex-shrink: 0;
+    display: flex;
+    flex-direction: column;
```

Comment:
Do we need to do the same for input-preview?

Reply:
Probably good to keep them consistent, these styles are duplicated, definitely something we could reduce. Updated the styles to match these. Would you like it if I also made a `PreviewDocument` component in `compass-aggregations` that they both use? Could do it another time also and prioritize this fix.

Reply:
I would probably say let's keep it as it is right now purely due to most of this going away at some point 😄 If we ever want to try to add a virtual list here though before that happens that might be a good reason to unify these two maybe?